pkg/controller/controller.go

```diff
@@ -98,8 +98,6 @@ func (c *Controller) CreateCluster(cluster model.Cluster, assets model.Assets) e
 	if cluster.Labels == nil {
 		cluster.Labels = model.Labels{}
 	}
-	// Set default cluster labels.
-	cluster.Labels[constants.ClusterNameLabelKey] = cluster.Name
 
```

Comment:
Was this a bug or is it tied to the subsequent work? The keto-token deployment is currently tied to a label existing with the cluster name so this may break?